src/opnsense/scripts/firmware/bogons.sh

```diff
@@ -73,10 +73,10 @@ ENTRIES_MAX=`pfctl -s memory | awk '/table-entries/ { print $4 }'`
 ENTRIES_TOT=`pfctl -vvsTables | awk '/Addresses/ {s+=$2}; END {print s}'`
 ENTRIES_V4=`pfctl -vvsTables | awk '/-\tbogons$/ {getline; print $2}'`
 LINES_V4=`wc -l ${WORKDIR}/fullbogons-ipv4.txt | awk '{ print $1 }'`
+
 if [ $ENTRIES_MAX -gt $((2*ENTRIES_TOT-${ENTRIES_V4:-0}+LINES_V4)) ]; then
-    # private and pseudo-private networks will be excluded
-    # as they are being operated by a separate GUI option
-    egrep -v "^100.64.0.0/10|^192.168.0.0/16|^172.16.0.0/12|^10.0.0.0/8" ${WORKDIR}/fullbogons-ipv4.txt > ${DESTDIR}/bogons
+    update_bogons fullbogons-ipv4.txt bogons 100.64.0.0/10 169.254.0.0/16 192.168.0.0/16 172.16.0.0/12 10.0.0.0/8
```

Comment:
```suggestion
    update_bogons fullbogons-ipv4.txt bogons 10.0.0.0/8 172.16.0.0/12 192.168.0.0/16 127.0.0.0/8 100.64.0.0/10 169.254.0.0/16  
```

reorder by type, add 127/8 even if it isn't in the bogons for consistency